core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -572,4 +572,24 @@ private[spark] object UIUtils extends Logging {
   def buildErrorResponse(status: Response.Status, msg: String): Response = {
     Response.status(status).entity(msg).`type`(MediaType.TEXT_PLAIN).build()
   }
+
+  def getHeaderNameWithTooltips(): Map[String, String] = {
+    val headerNameWithTooltips: Map[String, String] = Map("ID" -> "Execution ID of the SQL query.",
```

Comment:
I think this is not the appropriate place to put the tool tip info that is specific to the SQL page. May be you can put this function into `AllExecutionPage.scala`?

Reply:
@shahidki31 ,File name is UIUtils that's by i kept there and i want to use in AllExecutionsPageSuite.scala also but still need to change i will change it to AllExecutionPage.

Reply:
@shahidki31 , please confirm this , i think this is ok (because file is UIUtils.scala)

Reply:
Yes, I think we should move to AllExecutionPage.scala. May be you can mock the `ExecutionPagedTable` and make the function `private[ui]` and can access if necessary.

Reply:
ok shahidki31

Reply:
I think, you might need just to check if tooltips are populating or not. I don't think we need check each tooltip is matching or not.

Reply:
Or you can follow the approach of https://github.com/apache/spark/commit/3ea9d6825b64f0f21f066b4f07589c7d2c81bcbd instead of writing a method?

Reply:
ok, I will follow the approach of #25775  (because In SQL Tab based on the condition they will display header)